geo-types/src/multi_line_string.rs

```diff
@@ -81,3 +81,116 @@ impl<T: CoordinateType> IntoIterator for MultiLineString<T> {
         self.0.into_iter()
     }
 }
+
+impl<'a, T: CoordinateType> IntoIterator for &'a MultiLineString<T> {
```

Comment:
FYI: the implementation and tests were more or less copy+pasted across MultiLineString, MultiPolygon, and MultiPoint.